build.sbt

```diff
@@ -183,8 +183,14 @@ lazy val `sbt-metals` = project
       }
     },
     libraryDependencies --= libraryDependencies.in(ThisBuild).value,
-    scalacOptions --= Seq("-Yrangepos", "-Ywarn-unused-import")
+    scalacOptions --= Seq("-Yrangepos", "-Ywarn-unused-import"),
+    buildInfoPackage := "scala.meta.internal.sbtmetals",
+    buildInfoKeys := Seq[BuildInfoKey](
+      "metalsVersion" -> version.value,
```

Comment:
where is this used?

Reply:
answering myself: in #351

Reply:
Just a habit, generally useful to have although not used atm